util/reggen/reg_top.sv.tpl

```diff
@@ -563,13 +563,16 @@ ${reg_hdr}
         flds_no_we |= (not f.swaccess.needs_we()) << f_idx
       if flds_no_we != 0:
         flds_we_masked = f"&({sr_name}_flds_we | {len(sr.fields)}'h{flds_no_we:x})"
+        unused_flds_we_masked = f"|({sr_name}_flds_we & {len(sr.fields)}'h{flds_no_we:x})"
```

Comment:
did this work? typically we use a XOR since that prevents that individual bits mask each other...

Reply:
Yup, kokoro is green!

Reply:
But this is just the one bit case. I can change it back to unary XOR.

Reply:
Yeah that would be better. `^` should universally work.